local-notifications/android/src/main/java/com/capacitorjs/plugins/localnotifications/LocalNotificationSchedule.java

```diff
@@ -37,16 +37,19 @@ public LocalNotificationSchedule() {}
     private void buildEveryElement(JSObject schedule) {
         // 'year'|'month'|'two-weeks'|'week'|'day'|'hour'|'minute'|'second';
         this.every = schedule.getString("every");
+        if (this.every != null) {
+            this.repeats = true;
+        }
     }
 
     private void buildCountElement(JSObject schedule) {
         this.count = schedule.getInteger("count", 1);
     }
 
     private void buildAtElement(JSObject schedule) throws ParseException {
-        this.repeats = schedule.getBool("repeats");
         String dateString = schedule.getString("at");
         if (dateString != null) {
+            this.repeats = schedule.getBool("repeats");
```

Comment:
Does moving this inside the if check on `at` prevent the `repeats` flag being set for notifications that are built with the `on` details in `buildOnElement`? I noticed these two methods are called sequentially but for some reason the repeats flag was in here.